bl.c

```diff
@@ -224,13 +259,15 @@ jump_to_app()
 	systick_interrupt_disable();
 	systick_counter_disable();
 
-	/* and set a specific LED pattern */
-	led_off(LED_ACTIVITY);
-	led_on(LED_BOOTLOADER);
```

Comment:
I have now removed this setting of the LEDs prior to the jump, since I am deinitialising them in the board_deinit function.